templates/rhsm-katello-reconfigure.erb

```diff
@@ -63,6 +63,16 @@ else
   sed -i "s/baseurl.*/&\n\n$full_refresh_config/g" $CFG
 fi
 
+# also add the katello ca cert to the system wide ca cert store
+if [-d $CA_TRUST_ANCHORS]; then
+  cp %%(ca_cert_dir)skatello-server-ca.pem $CA_TRUST_ANCHORS
+  update-ca-trust
+
+  #restart if docker service is installed
+  service docker status >/dev/null && \
+    service docker restart >/dev/null 2&>1
+fi
+
 # restart goferd if it is installed and running
```

Comment:
I think service redirects to systemctl as a deprecated action, need to check if it will be removed in the future.